### PR TITLE
Add note about updating pip

### LINKED ISSUE #39120: OSX Install failure
Installing for OSX recommendations from the [Installation Guide](http://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html?#latest-releases-via-pip) failed.  Installing pip worked, but then pip was not able to complete the fetch.

##### ISSUE TYPE
 - Bug Report

##### INSTALLING LATEST ANSIBLE VERSION
```
$ sudo -H pip install ansible
Collecting ansible
  Could not fetch URL https://pypi.python.org/simple/ansible/: There was a problem confirming the ssl certificate: [SSL: TLSV1_ALERT_PROTOCOL_VERSION] tlsv1 alert protocol version (_ssl.c:590) - skipping
  Could not find a version that satisfies the requirement ansible (from versions: )
No matching distribution found for ansible
```
##### OS / ENVIRONMENT
Mac OS X El Capitan
10.11.6

##### STEPS TO REPRODUCE
```
sudo easy_install pip
sudo pip install ansible
```

##### EXPECTED RESULTS
I excepted Ansible to properly be downloaded via GIT method.

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY
Fixes: #39120

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
docs/docsite/rst/installation_guide/intro_installation.rst

##### ANSIBLE VERSION

```
2.7-devel
```

### CODE REVIEW

File: docs/docsite/rst/installation_guide/intro_installation.rst [hunk 1]
```diff
@@ -264,13 +264,18 @@ Then install Ansible with [1]_::
...
 
-    pip install git+https://github.com/ansible/ansible.git@devel
+   $ pip install git+https://github.com/ansible/ansible.git@devel
 
 If you are installing on OS X Mavericks, you may encounter some noise from your compiler.  A workaround is to do the following::
 
    $ sudo CFLAGS=-Qunused-arguments CPPFLAGS=-Qunused-arguments pip install ansible
 
-Readers that use virtualenv can also install Ansible under virtualenv, though we'd recommend to not worry about it and just install Ansible globally.  Do not use easy_install to install ansible directly.
+Readers that use virtualenv can also install Ansible under virtualenv, though we'd recommend to not worry about it and just install Ansible globally.  Do not use easy_install to install Ansible directly.
```
Comment (webknjaz) [inline docs/docsite/rst/installation_guide/intro_installation.rst]: > ` `..` `Do

rm double space here

File: docs/docsite/rst/installation_guide/intro_installation.rst [hunk 2]
```diff
@@ -264,13 +264,18 @@ Then install Ansible with [1]_::
...
+
+.. note::
+
+    Older versions of pip defaults to http://pypi.python.org/simple, which no longer works.
```
Comment (webknjaz) [inline docs/docsite/rst/installation_guide/intro_installation.rst]: I think, it's appropriate to refer to the `get-pip.py` doc here.
Also I think it's v9.3 + it's worth mentioning that it comes with newer TLS support bundled, which is crucial for MacOS users: https://mail.python.org/pipermail/python-announce-list/2018-April/011885.html

File: docs/docsite/rst/installation_guide/intro_installation.rst [hunk 3]
```diff
@@ -170,7 +170,7 @@ Add the following line to /etc/apt/sources.list:
 
 .. code-block:: bash
 
-    deb http://ppa.launchpad.net/ansible/ansible/ubuntu trusty main
+    $ deb http://ppa.launchpad.net/ansible/ansible/ubuntu trusty main
```
Comment (resmo) [inline docs/docsite/rst/installation_guide/intro_installation.rst]: this is a string in a file, not a bash command.

Comment (Akasurde) [reply docs/docsite/rst/installation_guide/intro_installation.rst]: my bad.

**Comment by Akasurde** [comment]
@resmo @webknjaz Done.

**Review by webknjaz — APPROVED** [review]
Great job!

**Review by bcoca — APPROVED** [review]

**Comment by Akasurde** [comment]
@resmo @webknjaz @bcoca Thanks for the review.

**Pull request merged**